public/app/features/dashboard-scene/edit-pane/ElementEditPane.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { SceneObject } from '@grafana/scenes';
+import { Stack, useStyles2 } from '@grafana/ui';
+import { OptionsPaneCategory } from 'app/features/dashboard/components/PanelEditor/OptionsPaneCategory';
+
+import { DashboardScene } from '../scene/DashboardScene';
+import { EditableDashboardElement, isEditableDashboardElement } from '../scene/types';
+
+import { DummySelectedObject } from './DummySelectedObject';
+
+export interface Props {
+  obj: SceneObject;
+}
+
+export function ElementEditPane({ obj }: Props) {
+  const element = getEditableElementFor(obj);
+  const categories = element.useEditPaneOptions();
+  const styles = useStyles2(getStyles);
+
+  return (
+    <Stack direction="column" gap={0}>
+      {element.renderActions && (
+        <OptionsPaneCategory
+          id="selected-item"
+          title={element.getTypeName()}
+          isOpenDefault={true}
+          className={styles.noBorderTop}
+        >
+          <div className={styles.actionsBox}>{element.renderActions()}</div>
+        </OptionsPaneCategory>
+      )}
+      {categories.map((cat) => cat.render())}
```

Comment:
Would this benefit from using `useMemo`?

Reply:
I missed `React.memo` wrapping `OptionsPaneCategory`. Yeah, no reason do memoize those yet.